crates/bevy_ecs/src/world/mod.rs

```diff
@@ -570,6 +570,7 @@ impl World {
 
     /// Returns an iterator of entities that had components of type `T` removed
     /// since the last call to [`World::clear_trackers`].
+    // TODO: return `std::iter::Copied` instead
```

Comment:
I think we should return a `std::iter::Copied` here but I don't know if it would be a breaking change or not